crates/uv-dispatch/src/lib.rs

```diff
@@ -36,6 +37,7 @@ pub struct BuildDispatch<'a> {
     config_settings: &'a ConfigSettings,
     source_build_context: SourceBuildContext,
     options: Options,
+    sdist_build_env_vars: FxHashMap<String, String>,
```

Comment:
This should be named something like `build_extra_env_vars`

Reply:
Done